examples/adaptive_crag/burr_docs/actions.txt

```diff
@@ -0,0 +1,166 @@
+Actions
```

Comment:
This is fine for now, I bet we could get it to query for the docs/download them all so we don't have to rely on them being updated :) That might be better as a TODO.

Reply:
Yeah, I agree. If we think of it as just _any_ data for this example, then it's fine.

Reply:
👍 yeah don't want to bog us down